cmd/lvmd/app/config.go

```diff
@@ -18,10 +18,13 @@ type Config struct {
 	DeviceClasses []*lvmdTypes.DeviceClass `json:"device-classes"`
 	// LvcreateOptionClasses are classes that define options for the lvcreate command
 	LvcreateOptionClasses []*lvmdTypes.LvcreateOptionClass `json:"lvcreate-option-classes"`
+	// ProfilingPort is the port number to expose profiling. If 0, profiling is disabled.
+	ProfilingPort int `json:"profiling-port"`
```

Comment:
how about changing this to a bind address instead so that people can not only control the port but also the network interface?

Reply:
All implementations have been moved to using a bind address instead.